### ISSUE TITLE
Token lifetime / Expiration

### ISSUE THREAD

**Issue by Schoof-T** [damienbod/Blazor.BFF.OpenIDConnect.Template#60]
I'm trying to understand the BFF pattern better and I have a question.

How do we check if the access token we are using is still valid? After a certain period we should be redirected to the login page, no?
How is this checked and performed?

Thanks

**Comment by damienbod** [comment]
Hi @Schoof-T When you get the AT back from the identity provider. a expired by property is also returned. You need to read this and set the cache to expire or implement the refresh process using this

Greetings Damien

**Comment by damienbod** [comment]
True, I wanted to keep the template basic, I should have an example somewhere, if not I will crate one, but I am busy and have a backlog infront of this

Greetings Damien
